### DISCUSSION TITLE
PrismaClient is unable to run in this browser environment (Next.js 14)

### DISCUSSION THREAD

**Discussion by Olivier-Muhunga** [prisma/prisma#23765] [Q&A]
### Bug description

Error: PrismaClient is unable to run in this browser environment, or has been bundled for the browser (running in `unknown`).

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

  database.bZ_USER.create({
    data: {
        email: 'user@email',
        password : '12345678',
        username: 'USER NAME',
  }
})

### Environment & setup

- OS: 
- Database: 
- Node.js version: 
-OS: macOS
-Database: MySQL
- Next.js ^14.1.4

### Prisma Version

```

prisma                  : 5.12.1
@prisma/client          : 5.12.1

```

**Comment by keinsell** [comment]
Prisma is not intended to be run in browser environment, you're supposed to run it in server-side environment and do not bundle it to client's code.

**Comment by jkomyno** [comment]
Hi @Olivier-Muhunga, I underline what @keinsell said. Also, please we'd need a minimal reproducible example from your side to be able to offer support. We recommend checking for your imports of `@prisma/client`, it should only be used in the `use 'server'` areas of your Next.js application. 

I will turn this issue into a discussion, so that my colleagues can help you further :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
